frontend/src/pages/EnvironmentSettings/EnvironmentSettings.style.tsx

```diff
@@ -0,0 +1,147 @@
+import Select from 'components/Select';
+import styled from 'styled-components';
+import { colorUsage, fontFamily, fontSize, fontWeight, getSpacing, lineHeight } from 'stylesheet';
+
+const Style = {
+  Container: styled.div`
+    display: flex;
+    flex-direction: column;
+    padding: 0 0 ${getSpacing(4)} 0;
+    margin: 0;
+    width: 800px;
```

Comment:
Is that `800px` value is used for all settings pages? could it be put inside the `stylesheet`?